CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing
+
+Thank you for your interest in contributing! Join the [**Plugin Marketplace channel**](https://community.mattermost.com/core/channels/plugins-marketplace) on the Mattermost community server for discussion about this plugin.
+
+## Reporting Issues
+
+If you think you found a bug, [please use the GitHub issue tracker](https://github.com/mattermost/mattermost-marketplace/labels/issues/new) to open an issue.
+
+## Community Plugin
+
+To add your plugin to the Marketplace, please open [an issue using thus template](https://github.com/mattermost/mattermost-marketplace/issues/new?template=add_plugin.md). To update your plugin, please also open an issue [using this template](https://github.com/mattermost/mattermost-marketplace/issues/new?template=update_plugin.md).
+
+### Playbook
+
+This is a playbook used my Mattermost core committers.
+
+#### New plugin
+
+JIRA template:
+  - [PM](https://mattermost.atlassian.net/browse/MM-22224)
+  - [Dev](https://mattermost.atlassian.net/browse/MM-22221)
+  - [Security](https://mattermost.atlassian.net/browse/MM-22225)
+  - [QA](https://mattermost.atlassian.net/browse/MM-22223)
+
+After a new plugin has been submitted, the assignees of the issue posts the following message:
+```
+## Process checklist
+- [ ] Request review and create JIRA tickets and link them here:
+    - PM:
+    - Dev:
+    - Security:
+    - QA:
+- [ ] Fork repo under Mattermost oranisation
+- [ ] Cut plugin release
+- [ ] Add release to Marketplace
+- [ ] Reach out on [Marketing Channel](https://community-release.mattermost.com/private-core/channels/marketing
+) to tweet about the plugin
+- [ ] Work with `@hanna.park` regarding swag
```

Comment:
Should we sync up with Hanna so we can be more explicit about expectations here?

Reply:
I'm working with Hanna on the details. For now, I think this can stay it's way.